bundles/rust/init.moon

```diff
@@ -0,0 +1,21 @@
+-- Copyright 2012-2016 The Howl Developers
```

Comment:
Sorry about that, I was only going with that @kirbyfan64 spoke upon in setting for the Copyright claim: https://github.com/howl-editor/howl/pull/257#discussion_r75767500

I'll change this!